stix/common/kill_chains.py

```diff
@@ -204,7 +245,102 @@ class KillChainPhasesReference(stix.EntityList):
     _binding_var = "Kill_Chain_Phase"
     _inner_name = "kill_chain_phases"
 
+    def _is_lmco(self, value):
+        return any(value.phase_id == x.phase_id for x in LMCO_KILL_CHAIN_PHASES)
+
+    def _fix_value(self, value):
+        if isinstance(value, KillChainPhase) and self._is_lmco(value):
```

Comment:
I think so, because we don't know what `KillChain` other `KillChainPhase` objects are a part of (without modifying other code to establish that relationship during `insert()`). We do, however, always know that an LMCO `KillChainPhase` is a part of the LMCO `KillChain`.